android/build.gradle

```diff
@@ -22,13 +22,16 @@ if(file.exists()) {
     smoothstreaming = json.smoothstreaming ?: smoothstreaming
 }
 
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 android {
-    compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+    compileSdkVersion safeExtGet("compileSdkVersion", 27)
 
     defaultConfig {
-        minSdkVersion 16 // RN's minimum version
-        targetSdkVersion 27
+        minSdkVersion safeExtGet('minSdkVersion', 16) // RN's minimum version
+        targetSdkVersion safeExtGet("compileSdkVersion", 27)
```

Comment:
How about the support library? It needs to match the compleSdkVersion, right?

Reply:
Honestly, I'm not sure. I assumed the support library could be top-level only.

Reply:
I'll go ahead and add it

Reply:
Added. I didn't mean to set `compileSdkVersion` for `targetSdkVersion`. Changed that to `targetSdkVersion` and used `supportLibVersion` for support libraries.